docs/configuration.md

```diff
@@ -0,0 +1,186 @@
+- `repodata`: Discover packages using repodata.json
+
+```toml
+package_discovery = "channeldata"
+```
+#### Artifact Discovery
+
+`artifact_discovery`: Determines the method for discovering package artifacts in the channel. Accepts two options:
+- `anaconda`: Discover artifacts using the Anaconda API. This only works if the package is hosted on Anaconda and the channel is public.
+- `repodata`: Discover artifacts using repodata.json
+```toml
+artifact_discovery = "anaconda"
+```
+
+#### Arch Subdirectory (Platform) Discovery
+
+`arch_subdir_discovery`: Determines how to discover all available architectures. Possible choices are:
+- `all`: Try all well-known architectures.
+- `channeldata`: Use channeldata.json to discover architectures.
+- A list of architectures to try within a `subdirs` key.
+```toml
+arch_subdir_discovery = "all"
+# or
+arch_subdir_discovery = { subdirs = ["linux-64", "osx-64", "win-64"] }
+```
+
+#### Repodata Patches
+  
+`repodata_patches_package`: Specifies a package that includes patches for "repodata". It is expected to be available in the channel.
+```toml
+repodata_patches_package = "conda-forge-repodata-patches"
+```
+
+#### Package Name to Feedstock Mapping
+
+`map_conda_forge_package_to_feedstock`: This is a flag used for enabling package name to feedstock name mapping for the `conda-forge` channel.
+If false, the package name is used as the feedstock name. If true, conda-forge's [feedstock-outputs](https://github.com/conda-forge/feedstock-outputs)
+is used to map the package name to the feedstock name.
+```toml
+map_conda_forge_package_to_feedstock = true
+```
+
+#### Provenance URL Pattern
+
+`provenance_url_pattern`: This provides a URL pattern to link to the source of a package. The `{feedstock}` placeholder is replaced with the feedstock name.
+```toml
+provenance_url_pattern = "https://github.com/conda-forge/{feedstock}-feedstock"
+```
+
+#### Package Filter
+
+`package_filter`: This property provides mechanism to filter packages by name or prefix.
+```toml
+# allows packages with names "r", "rpy2", "rstudio" and prefixes "r-", "_r-", "mro-"
+package_filter = { allowed_names = ["r", "rpy2", "rstudio"], allowed_prefixes = ["r-", "_r-", "mro-"] }
+```
+
+#### Broken Label Support
+
+`supports_broken_label`: This flag indicates that the channel supports a label called "broken" for releases that are not working (yanked).
+```toml
+supports_broken_label = true
+```
+
+#### Dashboards
+
+`dashboards`: List of dashboards that should be linked from the package browser. Dashboards are defined in the global `dashboards` section (see below).
+```toml
+dashboards = ["anaconda", "ghcr"]
+```
+
+#### Authentication
+
+`auth_username`, `auth_password`, `auth_quetz_token`, `auth_bearer_token`: Use these to configure authentication for the channel.
+
+```toml
+# HTTP Basic Authentication (for Artifactory)
+auth_username = "username"
+auth_password = "password"
+# or
+# Quetz Token Authentication
+auth_quetz_token = "token"
```

Comment:
Yeah, the idea behind naming it differently was to avoid confusion about the `/t/<token>` URL support. As far as I understand, the conda_token from pixi works with both Quetz and private Anaconda channels, this only works with Quetz